packages/telnyx_webrtc/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_webrtc: ^0.12.8
+  flutter_webrtc: 0.12.8
```

Comment:
Is there a reason we're hardcoding this specific value instead of allowing minor version updates automatically? Do we need to use this version?